test/recipes/80-test_cms.t

```diff
@@ -607,17 +607,17 @@ my @smime_cms_param_tests = (
     ],
 
     [ "enveloped content test streaming S/MIME format, ECDH, AES-128-CBC, SHA256 KDF",
-      [ "{cmd1}", @prov, "-encrypt", "-in", $smcont,
```

Comment:
Can you explain why this is failing? It looks like this would use ecdh_exch.c which looks like it uses X963 (which should work with SHA256).

Reply:
X963 should work with SHA256.

However, as per #24004 (and #19338), it use SHA-1 in fact although we specify `-keyopt ecdh_kdf_md:sha256`.